docs/nodes/nodes-library/nodes/Mailgun/README.md

```diff
@@ -0,0 +1,39 @@
+---
+permalink: /nodes/n8n-nodes-base.mailgun
+---
+
+# Mailgun
+
+[Mailgun](https://www.mailgun.com/) is a developer-friendly email sending platform that provides API-based email services that are easy to use.
+
+::: tip 🔑 Credentials
+The instructions for authenticating with Mailchimp are [here](../../../credentials/Mailgun/README.md).
```

Comment:
This says Mailchimp. Also, seems like the copy here and in Mailchimp (and some other ones?) are different from the rest of the nodes. Can you please check to make sure that everything is consistent?